crates/api/src/v1/mod.rs

```diff
@@ -28,13 +28,7 @@ pub enum ContentSource {
         /// The URL of the content.
         url: String,
         /// Optional, server accepts for HTTP Range header.
-        /// TODO remove rename, see issue: https://github.com/bytecodealliance/registry/issues/221
-        #[serde(
-            default,
-            skip_serializing_if = "is_false",
-            rename = "accept_ranges",
-            alias = "acceptRanges"
```

Comment:
This one is technically breaking "forward compatibility" since 0.3.0 code is still serializing as `accept_ranges`. Better to just swap the `rename` and `alias` values now and drop the `alias` later.